server/models/ipfspins.ts

```diff
@@ -0,0 +1,42 @@
+import * as Sequelize from 'sequelize';
+import { Model, DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type IpfsPinsAttributes = {
+  id: number;
+  address_id: number;
+  ipfs_hash: string;
+  created_at: Date;
+  updated_at: Date;
+};
+
+export type IpfsPinsInstance = ModelInstance<IpfsPinsAttributes>;
+
+export type IpfsPinsModelStatic = ModelStatic<IpfsPinsInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): IpfsPinsModelStatic => {
+  const IpfsPins = <IpfsPinsModelStatic>sequelize.define(
+    'IpfsPins',
+    {
+      id: { type: dataTypes.INTEGER, primaryKey: true },
```

Comment:
Technically should have `autoIncrement` set here as well, but wont break anything as it is.

Reply:
added